### ISSUE TITLE
Complete these tasks after deploying 0.5.2

### ISSUE THREAD

**Issue by benlk** [INN/Largo#821]
These tasks require and are required by various child themes' responses to 0.5.2. 

If you have a task, please add it below.

**Comment by benlk** [comment]
- [ ] update MWEN with 1col footer widget option
- [ ] update NPQ with 4col footer and a text widget: `<a href="<?php echo(of_get_option('subscribe_link')); ?>"><img id="header-subscribe" src="<?php echo(of_get_option('subscribe_image')); ?>" alt="Subscribe to Nonprofit Quarterly"></a>`
- [ ] remove SE Energy news social icons widget from footer-1

**Comment by benlk** [comment]
- [ ] Tell sites with custom footers that they'll lose those in the future. [Info about themes](https://github.com/INN/Largo/pull/820#issuecomment-128112430)
    - [ ] Catalyst Chicago and Chicago Reporter, who use this footer: https://cloud.githubusercontent.com/assets/1754187/9095562/e99cbcb0-3b85-11e5-804a-8b8b785953de.png
    - [ ] Connecticut Mirror's ctviewpoints and trendct, who will lose the six-column footer

( @aschweigert )

**Comment by rnagle** [comment]
- [ ] Since we're removing `inc/ad-codes.php`, we'll need to make sure any sites still using ACM for ads have been switched over to the DFP plugin after deployment.

**Comment by aschweigert** [comment]
We have some work to do on the voice of oc theme, the main thing is that fixing the bug where we were loading the homepage styles on every page somehow broke all of their css (because it was, somehow, inexplicably all contained in the stylesheet for their custom homepage layout). As a temporary fix, I put the CSS that used to be there in the main stylesheet directly but there are still some minor issues probably related to other css changes we've made since whenever that was last compiled. You'll notice also that the LESS file references a number of LESS files from Largo, but does not include the correct file paths...so I'm wondering if, perhaps, we don't even have the LESS files that were referenced here. The compiled output is definitely markedly different than even the old version that I restored temporarily. Also, there seem to be some issues with their custom header/navigation that we'll need to look into. This needs to be done today, preferably before I get an irate phone call from them.
